tests/test_umbral.py

```diff
@@ -18,15 +19,43 @@ def test_decapsulation_by_alice():
     priv_key = pre.gen_priv()
     pub_key = pre.priv2pub(priv_key)
 
-    sym_key, capsule = pre.encapsulate(pub_key)
+    sym_key, capsule = pre._encapsulate(pub_key)
     assert len(sym_key) == 32
 
     # The symmetric key sym_key is perhaps used for block cipher here in a real-world scenario.
 
-    sym_key_2 = pre.decapsulate_original(priv_key, capsule)
+    sym_key_2 = pre._decapsulate_original(priv_key, capsule)
     assert sym_key_2 == sym_key
 
 
+@pytest.mark.parametrize("N,threshold", parameters)
+def test_simple_api(N, threshold):
+    params = umbral.UmbralParameters()
+    pre = umbral.PRE(params)
+
+    priv_key_alice = keys.UmbralPrivateKey.gen_key(params)
+    pub_key_alice = priv_key_alice.get_pub_key(params)
+
+    priv_key_bob = keys.UmbralPrivateKey.gen_key(params)
+    pub_key_bob = priv_key_bob.get_pub_key(params)
+
+    plain_data = b'attack at dawn'
+    enc_data, capsule = pre.encrypt(pub_key_alice, plain_data)
```

Comment:
So, we really don't want to allow the user to optionally use their own symmetric tooling here?  I think this is a *big* decision decision to be made without discussion.

Although, maybe it makes sense to do it this way for now and add the option later if we want?

Reply:
I think, this can eventually be handled by having a default dem as a class attribute (well, at least on the basic level).
E.g.: `dem = UmbralDEM(...)` ->
```
class PRE(object):
    ...
    DEM = UmbralDEM
```
 ... `dem = PRE.DEM(...)`

Reply:
It would probably be for the best, currently, if the user can't easily extend their own DEM. Umbral is defined to use Salsa20-Poly1305, but we'll be switching to ChaCha20-Poly1305 to allow for us to add in authenticated data for more security in the overall system.

For that reason, users really shouldn't try to roll their own crypto here.

Reply:
Well yeah, I think this adds +1 to having `cls.DEM`, but will wait on @jMyles opinion

Reply:
I prefer for (in fact, I think that it only makes sense for) PRE to be a module, not a class.  So maybe instead we have:

`pre.set_dem_interface(MyCustomDEMClass)`

Reply:
We really need the KMS to have one standard DEM method. There isn't any good reason to have to make anything more complex when we have figured the one great way to do it.

Reply:
@tuxxy: Oh yeah, I think we all agree there.  I think this concern is more about making sure we have flexibility to build things other than the KMS.  

The way you've written this, I think we can probably just add an additional interface later without too much trouble.  So I think we can actually just move with this as-is.